src/utils/winner.js

```diff
@@ -0,0 +1,10 @@
+const winner = (RACE) => {
```

Comment:
```suggestion
const getWinners = (RACE) => {
```
`winner`라는 함수 이름이 함수보다는 변수처럼 정적인 부분에 어울리는 것 같아서 `getWinners`와 같이 수정해봐도 괜찮을 것 같아요!

Reply:
항상 우혁님 변수명에 있어서 잘 지으시는  것 같아요 ~ 배워갑니당